src/components/courses/LectureEditor.tsx

```diff
@@ -0,0 +1,112 @@
+import {ApiAddLecture, Lecture, Subcourse} from "../../types/Course";
+import React, {useState} from "react";
+import { Tag, Typography, DatePicker, TimePicker, message, Button, Select } from 'antd';
+import {CloseOutlined, UndoOutlined, PlusOutlined} from "@ant-design/icons";
+import moment, {Moment} from 'moment'
+import locale from 'antd/lib/calendar/locale/de_DE';
+import {Instructor} from "../../api/useInstructors";
+
+const { Text } = Typography;
+
+function lectureTimeToString (l: Lecture | ApiAddLecture){
+    const date = moment(l.start).format("DD.MM.YY");
+    const startTime = moment(l.start).format("HH:mm");
+    const endTime = moment(l.start).add(l.duration, "minutes").format("HH:mm");
+
+    return (`${date} ${startTime} - ${endTime}`)
+}
+
+export default function ({currentLectures, newLectures, setNewLectures, removeLectures, setRemoveLectures, subcourse, instructors }: { currentLectures: Lecture[], newLectures: ApiAddLecture[], setNewLectures(newLectures: ApiAddLecture[]): void, removeLectures: Lecture[], setRemoveLectures(oldLectures: Lecture[]): void, subcourse: Subcourse, instructors: Instructor[] }) {
```

Comment:
Although function expressions can be anonymous, it makes it easier to find errors in stacktraces if the stacktrace contains function names, so I'd go with `function LectureEditor(...)` here